lib/mayaUsd/ufe/UsdAttributes.cpp

```diff
@@ -504,28 +504,37 @@ bool UsdAttributes::canRenameAttribute(
 
 static void setConnections(
     const PXR_NS::UsdPrim& prim,
-    const SdfPath&         OldPropertyPath,
+    const SdfPath&         oldPropertyPath,
     const SdfPath&         newPropertyPath)
 {
-    // Update the connections with the new attribute name.
-    for (const auto& node : prim.GetChildren()) {
-        for (auto& attribute : node.GetAttributes()) {
-            PXR_NS::UsdAttribute  attr = attribute.As<PXR_NS::UsdAttribute>();
-            PXR_NS::SdfPathVector sources;
-            attr.GetConnections(&sources);
-            bool hasChanged = false;
-            // Check if the node attribute is connected to the original property path.
-            for (size_t i = 0; i < sources.size(); ++i) {
-                if (sources[i] == OldPropertyPath) {
-                    sources[i] = newPropertyPath;
-                    hasChanged = true;
-                }
-            }
-            // Update the connections with the new property path.
-            if (hasChanged) {
-                attr.SetConnections(sources);
+    for (auto& attribute : prim.GetAttributes()) {
+        PXR_NS::UsdAttribute  attr = attribute.As<PXR_NS::UsdAttribute>();
+        PXR_NS::SdfPathVector sources;
+        attr.GetConnections(&sources);
+
+        bool hasChanged = false;
+        // Check if the node attribute is connected to the original property path.
+        for (size_t i = 0; i < sources.size(); ++i) {
+            if (sources[i] == oldPropertyPath) {
+                sources[i] = newPropertyPath;
+                hasChanged = true;
             }
         }
+        // Update the connections with the new property path.
+        if (hasChanged) {
+            attr.SetConnections(sources);
+        }
+    }
+}
+
+static void setConnectionsWithChildren(
```

Comment:
I find the name ambiguous. It could be:
- `setConnectionsOfAllChildren()`
- `updateChildrenConnections()` +1